.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+#### Please paste your output of croutonversion from chroot below:
+```
+Replace this line with your output.
+```
+
+#### Please describe your issue:
+
+
+#### Please describe how to reproduce your issue(if known):
```

Comment:
Maybe go as far as using 'sudo edit-chroot -all' in case their chroots are encrypted, the extra 'l' doesn't hurt anything if it's not encrypted.